hwdb.d/60-evdev.hwdb

```diff
@@ -233,6 +233,13 @@ evdev:name:AlpsPS/2 ALPS DualPoint TouchPad:dmi:bvn*:bvr*:bd*:svnDellInc.:pnLati
  EVDEV_ABS_35=76:1815:22
  EVDEV_ABS_36=131:1330:30
 
+# Dell Latitude E5510
```

Comment:
nit: the listings are sorted on the model name/number. IE: move this one section up, before E6220.